pandas/core/util/hashing.py

```diff
@@ -55,7 +55,11 @@ def _combine_hash_arrays(arrays, num_items):
 
 
 def hash_pandas_object(
-    obj, index=True, encoding="utf8", hash_key=None, categorize=True
+    obj,
+    index: bool = True,
+    encoding: str = "utf8",
+    hash_key=None,
+    categorize: bool = True,
```

Comment:
adding type hints here is causing `pandas/core/util/hashing.py:132: error: Incompatible types in assignment (expression has type "chain[Any]", variable has type "Generator[Any, None, None]")` further down as the body is now checked.

Reply:
it looks like the conda environment now has mypy 0.720. result!

fix is here https://github.com/pandas-dev/pandas/pull/28339/commits/304351e23d1586e0cc28108aff9a2a8a9fac5bf7 is you want to include in the PR. (can't use py3.6 variable annotations though.)